state_formatters/dp_formatters.py

```diff
@@ -263,13 +273,24 @@ def property_extraction_formatter_dialog(dialog: Dict) -> List[Dict]:
 
 
 def property_extraction_formatter_last_bot_dialog(dialog: Dict) -> List[Dict]:
-    if dialog["bot_utterances"]:
-        dialog_history = [dialog["bot_utterances"][-1]["text"]]
+    dialog = utils.get_last_n_turns(dialog, bot_last_turns=1)
+    dialog = utils.replace_with_annotated_utterances(dialog, mode="punct_sent")
+    if len(dialog["utterances"]) >= 2:
+        dialog_history = [dialog["utterances"][-2]["text"]]
```

Comment:
should it be last humn uttr? I do not get what utterance you are trying to get to history

Reply:
@zucchini-nlp

Reply:
should be last bot utterance, fixed it